flixel/system/debug/console/Console.hx

```diff
@@ -117,8 +117,10 @@ class Console extends Window
 		// openfl/openfl#1856
 		FlxG.stage.addEventListener(KeyboardEvent.KEY_DOWN, function(e:KeyboardEvent)
 		{
-			if (FlxG.debugger.visible && FlxG.game.debugger.console.visible && e.keyCode == Keyboard.TAB)
+			if (FlxG.debugger.visible && FlxG.game.debugger.console.visible && e.keyCode == Keyboard.TAB) {
 				FlxG.stage.focus = input;
+			}
+			e.preventDefault();
```

Comment:
do we want to preventDefault on all keys, or just on tab? should we prevent it on all targets? does this issue happen on all targets?

I'm thinking "no" for all 3